src/Parser/DeathKnight/Blood/Modules/Talents/Heartbreaker.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+const RP_RESOURCE_ID = 6;
```

Comment:
prefered to use `RESOURCE_TYPES.RUNES;` (Import `import RESOURCE_TYPES from 'common/RESOURCE_TYPES';`)